test/gerkin/tests/features/data_update_entries.feature

```diff
@@ -0,0 +1,15 @@
+Feature: API endpoint allowing users to update multiple records in the Digital Registries database.
+  Request endpoint: PUT /data/{registryname}/{versionnumber}/update-entries
+
+  Background:
+    Given The user wants to update multiple records in the Digital Registries database
```

Comment:
Not sure about this given statement. This description of a `Given` is closer to the scenario description than actual setup/system requirements. It seems to be the case in other features as well, can we change it?